qovery/resource_job.go

```diff
@@ -0,0 +1,511 @@
+				Computed: true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					modifiers.NewInt64DefaultModifier(int64(job.DefaultCPU)),
+				},
+				Validators: []tfsdk.AttributeValidator{
+					validators.Int64MinValidator{Min: int64(job.MinCPU)}, // TODO(benjaminch): useless check, by design won't be < 0
+				},
+			},
+			"memory": {
+				Description: descriptions.NewInt64MinDescription(
+					"RAM of the job in MB [1024MB = 1GB].",
+					int64(job.MinMemory),
+					pointer.ToInt64(int64(job.DefaultMemory)),
+				),
+				Type:     types.Int64Type,
+				Optional: true,
+				Computed: true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					modifiers.NewInt64DefaultModifier(int64(job.DefaultMemory)),
+				},
+				Validators: []tfsdk.AttributeValidator{
+					validators.Int64MinValidator{Min: int64(job.MinMemory)},
+				},
+			},
+			"max_duration_seconds": {
+				Description: descriptions.NewInt64MinDescription(
+					"Job's max duration in seconds.",
+					int64(job.MinDurationSeconds),
+					pointer.ToInt64(int64(job.DefaultMaxDurationSeconds)),
+				),
+				Type:     types.Int64Type,
+				Optional: true,
+				Computed: true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					modifiers.NewInt64DefaultModifier(int64(job.DefaultMaxDurationSeconds)),
+				},
+				Validators: []tfsdk.AttributeValidator{
+					validators.Int64MinValidator{Min: int64(job.MinDurationSeconds)}, // TODO(benjaminch): useless check, by design won't be < 0
+				},
+			},
+			"max_nb_restart": {
+				Description: descriptions.NewInt64MinDescription(
+					"Job's max number of restarts.",
+					int64(job.MinNbRestart),
+					pointer.ToInt64(int64(job.DefaultMaxNbRestart)),
+				),
+				Type:     types.Int64Type,
+				Optional: true,
+				Computed: true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					modifiers.NewInt64DefaultModifier(int64(job.DefaultMaxNbRestart)),
+				},
+				Validators: []tfsdk.AttributeValidator{
+					validators.Int64MinValidator{Min: int64(job.MinNbRestart)}, // TODO(benjaminch): useless check, by design won't be < 0
+				},
+			},
+			"port": {
+				Description: descriptions.NewInt64MinMaxDescription(
+					"Job's probes port.",
+					port.MinPort,
+					port.MaxPort,
+					nil,
+				),
+				Type:     types.Int64Type,
+				Optional: true,
+				Validators: []tfsdk.AttributeValidator{
+					validators.Int64MinMaxValidator{Min: port.MinPort, Max: port.MaxPort},
+				},
+			},
+			"auto_preview": {
+				Description: "Specify if the environment preview option is activated or not for this job.",
+				Type:        types.BoolType,
+				Optional:    true,
+				Computed:    true,
+			},
+			"schedule": {
+				Description: "Job's schedule.",
+				Required:    true,
+				Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+					"on_start": {
+						Description: "Job's schedule on start.",
+						Optional:    true,
+						Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+							"entrypoint": {
```

Comment:
entrypoint can be optional like arguments (it would be an empty string)

Reply:
Added `Computed:    true,` should fix it IMO, let's try